web/packages/shared/components/FieldSelect/FieldSelectCreatable.tsx

```diff
@@ -99,6 +106,8 @@ type CreatableProps = SelectCreatableProps & {
   autoFocus?: boolean;
   label?: string;
   rule?: (options: unknown) => () => unknown;
+  markAsError?: boolean;
+  ariaLabel?: string;
```

Comment:
i don't think we do aria labeling... should we be??

Reply:
The only reason for adding this was to query the element in test. This component is a bit pain to query otherwise. I don't think it would hurt though!